### DISCUSSION TITLE
ant design pro v6.0.0用不了umi小米饭

### DISCUSSION THREAD

**Discussion by Abengate** [umijs/umi#12114] [Q&A]
![image](https://github.com/umijs/umi/assets/122261591/a9672bf6-64f7-4d89-9c2f-93dcc0f5ec84)
![image](https://github.com/umijs/umi/assets/122261591/dc9f4ce8-68dc-47ce-83ae-0940c6f5aab4)
![image](https://github.com/umijs/umi/assets/122261591/03eb71ea-3cb3-4fab-b6c3-a2fec64c3182)

**Comment by fz6m** [comment]
See #12113
